int32 variable_num_
float64[] qes
int64 final_cost
int32 fo 
int32 ord 
int32 in
float64 interval 
float64 l1 
float64 l2 
float64 nl2 
float64 l3 
float64 mv 
float64 ma
float64 clearance
int32 size
geometry_msgs/Point[] points
geometry_msgs/Point[] base_point
geometry_msgs/Point[] direction
---
int32 result
int32 variable_num_
float64[] qes
int64 final_cost
int32 fo 
int32 ord 
int32 in
float64 interval 
float64 l1 
float64 l2 
float64 nl2 
float64 l3 
float64 mv 
float64 ma
geometry_msgs/Point[] points
geometry_msgs/Point[] base_point
geometry_msgs/Point[] direction
